web/src/components/editor/editor.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+	EditorProps,
+	Editor as ReactMonacoEditor,
+	type Monaco,
+	OnChange,
+} from "@monaco-editor/react";
+import { editor } from "monaco-editor";
+import clsx, { ClassValue } from "clsx";
+import { twMerge } from "tailwind-merge";
+
+function mergeCSSClasses(...inputs: ClassValue[]) {
+	return twMerge(clsx(inputs));
+}
```

Comment:
Can you move this function to `utils/tailwind.ts` and rename it to `cn`? I've seen this in some other repositories like that and seems to me like a best practice.